src/commands/apphosting-builds-create.ts

```diff
@@ -0,0 +1,42 @@
+import * as apphosting from "../gcp/apphosting";
+import { logger } from "../logger";
+import { Command } from "../command";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+
+function generateId(n = 6): string {
```

Comment:
Can this be moved out into a util file? I see you've written it 3 times now.

Reply:
Done!